gold-phone-input.html

```diff
@@ -49,6 +49,10 @@
   #label {
     left: 40px;
   }
+
+  input[is="iron-input"] {
+    min-width: 0;
```

Comment:
Why don't we fix this in `<paper-input-container>` where the rest of the input styling is normalized?